client/login/magic-login/handle-emailed-link-form-jetpack-connect.tsx

```diff
@@ -120,6 +139,28 @@ const HandleEmailedLinkFormJetpackConnect: FC< Props > = ( { emailAddress, token
 				] }
 			</h3>
 		</EmptyContent>
+	) : (
+		<Main className="magic-login">
+			{ renderJetpackLogo() }
+
+			<div className="magic-login__successfully-jetpack">
+				<h1 className="magic-login__form-header">{ translate( 'Email confirmed!' ) }</h1>
+				<p>
+					{ [
+						translate( 'Logging in as {{strong}}%(emailAddress)s{{/strong}}', {
+							args: {
+								emailAddress,
+							},
+							components: {
+								strong: <strong></strong>,
+							},
+						} ),
+						'...',
```

Comment:
This should be a part of the translated string as different languages have different characters for dots.

So, the string should be

```ts
'Logging in as {{strong}}%(emailAddress)s{{/strong}}…' // Note … at the end
```